.github/workflows/functional-tests-workflow.yml

```diff
@@ -27,6 +27,7 @@ jobs:
     runs-on: ubuntu-latest
     env:
       DBT_TEST_ATHENA_S3_STAGING_DIR: s3://dbt-athena-query-results-${{ inputs.aws-region }}
```

Comment:
given the fact that we use the same bucket as staging and for tmp tables, maybe we consider to use a prefix?
`s3://dbt-athena-query-results-${{ inputs.aws-region }}/staging/`